luigi/lock.py

```diff
@@ -65,12 +81,7 @@ def get_info(pid_dir, my_pid=None):
         my_pid = os.getpid()
 
     my_cmd = getpcmd(my_pid)
-
-    if six.PY3:
-        cmd_hash = my_cmd.encode('utf8')
-    else:
-        cmd_hash = my_cmd
-
+    cmd_hash = my_cmd.encode('utf8')
```

Comment:
Hmm, now this causes breakage on Linux, py27: `UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 4: ordinal not in range(128)`.

Would this change to [line 52](https://github.com/spotify/luigi/pull/2130/commits/c676395b4eeb1cf14b04aec95848faccfd1e2197#diff-6e699ca1b8953337c23f81c42066b0bbL52) be appropriate?

```patch
-return fh.read().replace('\0', ' ').rstrip()
+return fh.read().replace('\0', ' ').decode('utf8').rstrip()
```